### PR TITLE
home-page-added

### PR DESCRIPTION
**Pull request by vaibhav18matere**
Check for preview : https://deploy-preview-2--flamboyant-tesla-60f5a8.netlify.app/

### CODE REVIEW

File: homepage.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    position: relative;
+    -webkit-animation-name: slide_animation ;
+    -webkit-animation-duration: 15s;
+    -webkit-animation-iteration-count:infinite;
+    -webkit-animation-direction: alternate;
+    -webkit-animation-play-state:running ;
+}
+
+.img-container{
+    height: 1000px;
```
Comment (harshkotkar5557) [inline homepage.css:66]: You can remove commented part

**Review by harshkotkar5557 — APPROVED** [review]
Remove commented code otherwise everything is well and good.

**Review by manojasarsa — APPROVED** [review]
LGTM 👍

File: homepage.css (hunk 1 above)
Comment (vaibhav18matere) [reply homepage.css:66]: > You can remove commented part

thank you for bringing this into my notice, will do this!

**Review by aayushjain-code — COMMENTED** [review]
Try to give some padding left and right of navbar

**Review by aayushjain-code — APPROVED** [review]

**Pull request merged**